content/source/docs/enterprise/users-teams-organizations/teams.html.md

```diff
@@ -4,14 +4,83 @@ page_title: "Teams - Terraform Enterprise"
+
+Members of the owners team have [admin permissions](./permissions.html#admin) on every workspace in the organization. Additionally, the following tasks can only be performed by organization owners:
+
+- Creating new workspaces
+- Managing Sentinel policies
+- Overriding `soft mandatory` Sentinel policies
+- Creating and deleting teams
+- Managing team membership
+- Viewing the full list of teams, including teams the user does not belong to
+- Managing [organization settings][]
+
+## Managing Teams
+
+Teams are managed in the [organization settings][]. Click the "Settings" link in the top navigation bar, then click the "Teams" link in the left sidebar.
+
+![Screenshot: the teams page, displaying a list of teams. Each team's entry shows how many members it has.](./images/teams-list.png)
+
+The teams page includes a list of the organization's teams. Clicking a team's name in the list loads its team settings page, where you can manage its membership and settings:
+
+![Screenshot: a team's settings page](./images/teams-team-settings.png)
+
+The team settings page includes a list of the team's current members, with badges to indicate which users have [two-factor authentication](./2fa.html) enabled.
+
+Only organization owners can manage teams; other users can view the teams page in read-only mode. When a non-owner views the teams page, it only includes the current user's teams.
+
+### Creating and Deleting Teams
+
+-> **API:** See the [Teams API](../api/teams.html).
+
+Organization owners can create new teams from the teams page, using the controls under the "Create a New Team" header.
+
+To create a new team, provide a name (unique within the organization) and click the "Create team" button. Team names can include numbers, letters, underscores (`_`), and hyphens (`-`).
+
+To delete a team, go to the target team's settings page and click the "Delete TEAM NAME" button.
+
+### Managing Team Membership
+
+-> **API:** See the [Team Members API](../api/team-members.html).
+
+Organization owners can use a team's settings page to add and remove users from the team.
+
+The structure of your TFE teams should resemble your company's group structure.
+
+To add a user, enter their username in the "Username" text field (located under the "Add a New Team Member" header) and click the "Add member" button. (You must know the user's TFE username; users cannot be added using email addresses or other personal information.)
+
+To remove a user, click the "🗑" (trash can) button by their entry in the member list.
+
+### API Tokens
+
+-> **API:** See the [Team Tokens API](../api/team-tokens.html).
+
+Each team can have a special service account API token that is not associated with a specific user. You can manage this API token from a team's settings page. See [Service Accounts](./service-accounts.html) for more information.
```

Comment:
In this case I would say "from the team's settings page" because "each team" establishes a specific frame of reference. But again, do as you will!